### ISSUE TITLE
Show labels in the Summary view

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/did#461]
### TLDR ###
Show labels more prominently, at least in the Allocation view.

### Discussion ###

Labels is a great addition that was brought to the product with Boron. However, I think the labels are too hidden in the UX to relaease their full value potential. Specifically in the Allocation view (which is the best view to look at when registering hours in other systems), it would be very useful to show them next to the projects.

![image](https://user-images.githubusercontent.com/1837390/90862946-81c11f80-e38e-11ea-9661-807590021065.png)

We've discussed this before in #110  - is this a planned feature?

### Related issues ###
https://github.com/Puzzlepart/did365/issues/110#issuecomment-593856596
